packages/gateway-client/src/worker-messaging.ts

```diff
@@ -8,11 +8,13 @@ export enum ServerPeerStatus {
 export enum WorkerMessageType {
     SERVER_PEER_STATUS = 'SERVER_PEER_STATUS',
     LOADED_RELAYS = 'LOADED_RELAYS',
+    SW_HEARTBEAT = 'SW_HEARTBEAT',
```

Comment:
Let's try and avoid using abbreviations and acronyms in names:
```suggestion
    HEARTBEAT = 'SW_HEARTBEAT',
```